api/lib/oauth/oauth.js

```diff
@@ -116,11 +112,11 @@ module.exports = function(configFilePath) {
         "Content-Type": "application/json"
       }
     };
-    request(options, function(error1, response, body) {
-      if (error1) {
-        logger.error("Failed to get user info from UAA", { "userToken": userToken, "http-options": options, "error": error1 });
-        error1.statusCode = HttpStatus.INTERNAL_SERVER_ERROR;
-        callback(error1, null);
+    request(options, function(error, response, body) {
+      if (error) {
+        logger.error("Failed to get user info from UAAss", { "userToken": userToken, "http-options": options, "error": error });
```

Comment:
The error message has a typo, which is "UAAss"
How about using "Failed to get user info from UAA"?